[Docs] Error while building examples: tracking_quick_start.py

```
tracking_quick_start.py
Dataset is already in place. If you want to fetch it again please first remove the folder /home/bishakh/.dipy/stanford_hardi 
Dataset is already in place. If you want to fetch it again please first remove the folder /home/bishakh/.dipy/stanford_hardi 
/home/bishakh/projects/dipy/dipy/direction/peaks.py:562: RuntimeWarning: invalid value encountered in double_scalars
  return np.sqrt(numer / denom)
/home/bishakh/projects/dipy/dipy/direction/peaks.py:562: RuntimeWarning: invalid value encountered in double_scalars
  return np.sqrt(numer / denom)
/home/bishakh/projects/dipy/dipy/direction/peaks.py:562: RuntimeWarning: invalid value encountered in double_scalars
  return np.sqrt(numer / denom)
/home/bishakh/projects/dipy/dipy/direction/peaks.py:562: RuntimeWarning: invalid value encountered in double_scalars
  return np.sqrt(numer / denom)
Traceback (most recent call last):
  File "../tools/make_examples.py", line 122, in <module>
    exec(open(script).read(), namespace)
  File "<string>", line 101, in <module>
TypeError: slice indices must be integers or None or have an __index__ method
make: *** [Makefile:122: rstexamples-stamp] Error 1

```

--- a/doc/examples/brain_extraction_dwi.py
+++ b/doc/examples/brain_extraction_dwi.py
@@ -61,7 +61,7 @@ Quick view of the results middle slice using matplotlib.
 import matplotlib.pyplot as plt
 from dipy.core.histeq import histeq
 
-sli = data.shape[2] / 2
+sli = data.shape[2] // 2
 plt.figure('Brain segmentation')
 plt.subplot(1, 2, 1).set_axis_off()
 plt.imshow(histeq(data[:, :, sli].astype('float')).T,

--- a/doc/examples/reconst_dsi.py
+++ b/doc/examples/reconst_dsi.py
@@ -54,7 +54,7 @@ dsmodel = DiffusionSpectrumModel(gtab)
 Lets just use one slice only from the data.
 """
 
-dataslice = data[:, :, data.shape[2] / 2]
+dataslice = data[:, :, data.shape[2] // 2]
 
 dsfit = dsmodel.fit(dataslice)
 

--- a/doc/examples/reconst_dsi_metrics.py
+++ b/doc/examples/reconst_dsi_metrics.py
@@ -44,7 +44,7 @@ dsmodel = DiffusionSpectrumModel(gtab, qgrid_size=35, filter_width=18.5)
 Lets just use one slice only from the data.
 """
 
-dataslice = data[30:70, 20:80, data.shape[2] / 2]
+dataslice = data[30:70, 20:80, data.shape[2] // 2]
 
 """
 Normalize the signal by the b0

--- a/doc/examples/tracking_quick_start.py
+++ b/doc/examples/tracking_quick_start.py
@@ -96,7 +96,7 @@ which we will use as the basis to perform the tracking.
 
 ren = fvtk.ren()
 
-slice_no = data.shape[2] / 2
+slice_no = data.shape[2] // 2
 
 fvtk.add(ren, fvtk.peaks(csd_peaks.peak_dirs[:, :, slice_no:slice_no + 1],
                          stopping_values[:, :, slice_no:slice_no + 1]))

